pyproject.toml

```diff
@@ -43,111 +44,65 @@ docs = [
     "sphinxext-opengraph", # https://github.com/wpilibsuite/sphinxext-opengraph
 ]
 
-
 [project.scripts]
 py-projects = "py_launch_blueprint.projects:main"
 
 [build-system]
-requires = ["hatchling","hatch-vcs", "setuptools>=64", "setuptools-scm>=8"]
+requires = ["hatchling", "hatch-vcs", "setuptools>=64", "setuptools-scm>=8"]
 build-backend = "hatchling.build"
 
 [tool.setuptools_scm]
 write_to = "py_launch_blueprint/_version.py"
 fallback_version = "0.0.1"
 
 [tool.ruff]
-# Target Python Version
 target-version = "py310"
-
-# Line length
 line-length = 88
-
 lint.select = [
-    "E",   # pycodestyle errors
-    "F",   # pyflakes
-    "I",   # isort
-    "B",   # flake8-bugbear
-    "C4",  # flake8-comprehensions
-    "UP",  # pyupgrade
-    "N",   # pep8-naming
-    "RUF", # Ruff-specific rules
-    "W",   # pycodestyle warnings
-    "YTT", # flake8-2020
-    "S",   # flake8-bandit
+    "E", "F", "I", "B", "C4", "UP", "N", "RUF", "W", "YTT", "S"
 ]
-
-# Ignore specific rules
-lint.ignore = [
-    # "E501",  # line length violations
-]
-
-# Allow autofix behavior for specific rules
+lint.ignore = []
 fix = true
-lint.unfixable = []  # Rules that should not be fixed automatically
-
-# Exclude files/folders
+lint.unfixable = []
 exclude = [
-    ".git",
-    ".venv",
-    "__pycache__",
-    "build",
-    "dist",
-    "docs/source/conf.py"
+    ".git", ".venv", "__pycache__", "build", "dist", "docs/source/conf.py"
 ]
 
 [tool.ruff.pycodestyle]
 max-line-length = 88
 
-# Per-file-ignores
 [tool.ruff.lint.per-file-ignores]
 "__init__.py" = ["F401"]  # Ignore unused imports in __init__.py files
-"tests/*" = ["S101", "S105", "S106"]      # Ignore assert statements in tests
+"tests/*" = ["S101", "S105", "S106"]  #
 # S101: This check flags the use of assert statements. The reason is that assert statements are removed when Python code is optimized (run with the -O flag), which can lead to security vulnerabilities if assert is used for security checks.
-# S105: This generally flags hardcoded password strings in code.
```

Comment:
@Tatianah3  I am not going to specify every removed comment but please go back and add them all back.